packages/odyssey-react/src/components/Tabs/Tabs.module.scss

```diff
@@ -29,11 +29,30 @@
   color: var(--TabTextColor);
   font-family: inherit;
   font-size: var(--TabFontSize);
-  font-weight: var(--TabFontWeight);
 
   &:last-child {
     margin-inline-end: 0;
   }
+
+  &:hover,
+  &:focus {
+    color: var(--TabHoverTextColor);
+  }
+
+  &:focus {
+    outline: 0;
+  }
+}
+
+.label {
+  @include border-radius(var(--LabelBorderRadius));
+
+  .tab:focus & {
```

Comment:
Do we want to consider using `.tab:focus-visible ...` here to restrict the focus ring to keyboard navigation?  It seems odd to show the focus ring when clicking on a given tab.

Reply:
@kamalgill-okta thanks for raising this. I agree that the `:focus-visible` behavior feels more intuitive; however, I see a few hurdles for this PR:

- `:focus-visible` isn't available in all the browsers we support, so we'd need a polyfill or fallbacks for IE11 and (older) Safari
- We'd want this behavior to be uniform throughout the system to avoid confusing users with inconsistency 
- while WCAG criterion 2.4.7 refers to "keyboard focus", we'll want to ensure this change doesn't compromise on usability in another way (e.g. users with low digital literacy or low-vision users with pointer devices)

I've opened [this ticket](https://oktainc.atlassian.net/browse/OKTA-485470) to keep track of this. Feel free to leave any thoughts there or update my description!